src/main/java/org/ibp/api/rest/germplasm/GermplasmResource.java

```diff
@@ -396,4 +399,30 @@ public long getFilteredCount() {
 		return new ResponseEntity<>((updateExecuted) ? HttpStatus.OK : HttpStatus.NO_CONTENT);
 	}
 
+	@ApiOperation(value = "Merge duplicate germplasm into a single germplasm")
+	@PreAuthorize("hasAnyAuthority('ADMIN', 'GERMPLASM', 'MANAGE_GERMPLASM', 'MERGE_GERMPLASM')")
+	@RequestMapping(value = "/crops/{cropName}/germplasm/merge", method = RequestMethod.POST)
+	@ResponseBody
+	public ResponseEntity<Void> mergeGermplasm(@PathVariable final String cropName,
+		@RequestBody final GermplasmMergeRequestDto germplasmMergeRequestDto) {
+		this.germplasmService.mergeGermplasm(germplasmMergeRequestDto);
+		return new ResponseEntity<>(HttpStatus.OK);
+	}
+
+	@ApiOperation(value = "Get the list of duplicate germplasm that were merged into the specified germplasm")
+	@RequestMapping(value = "/crops/{cropName}/germplasm/merge/{gid}", method = RequestMethod.GET)
```

Comment:
I'm thinking it's better to rename endpoint as: 
`/crops/{cropName}/germplasm/{gid}/merges`
What do you think @clarysabel ?